docs/page-objects/testing.rst

```diff
@@ -322,3 +322,42 @@ The coverage for page object code is reported correctly if tools such as
 `coverage`_ are used when running web-poet tests.
 
 .. _coverage: https://coverage.readthedocs.io/
+
+Item adapters
+=============
+
+The testing framework uses the itemadapter_ library to convert items to dicts
+when storing them in fixtures and when comparing the expected and the actual
+output. As adapters may influence the resulting dicts, it's important to use
+the same adapter in both cases and it may be useful to use a different adapter
+in tests and in production (for example, you may want to omit empty fields in
+production but be able to distinguish between empty and absent fields in
+tests). For this you can set the ``adapter_type_name`` field in the metadata
+dictionary to the type name of a class that inherits from
```

Comment:
I think both here and in the section about `frozen_time` we should replace “the metadata dictionary” with “the metadata dictionary passed to Fixture.save()”, to give more of a hint on how to send that metadata dictionary. We could alternative link “metadata dictionary” to the Fixtures section above that explains that it is passed to Fixture.save().